test/token.test.ts

```diff
@@ -164,62 +158,152 @@ describe('token integration', () => {
       await tx.prove();
       await tx.send();
     });
+  });
```

Comment:
so are we going to get rid of initialize?

Reply:
Yes, I'm preparing a separate PR for that.